shared/actions/platform-specific/index.native.js

```diff
@@ -301,6 +302,10 @@ const copyToClipboard = (_: any, action: ConfigGen.CopyToClipboardPayload) => {
   Clipboard.setString(action.payload.text)
 }
 
+const handleFilePickerError = (state: TypedState, action: ConfigGen.FilePickerErrorPayload) => {
+  Alert.alert('Error', action.payload.error.message)
```

Comment:
Throw up an alert with 'Error' as the title, the message as the text (usually 'Permission denied'), and a single 'OK' button